src/themes/default/index.ts

```diff
@@ -15,7 +15,7 @@ export default (brandPrimary: string) => {
   const uiFontSize = 14;
   const colorBackground = legacyStyles.themeGrayLighter;
   const colorContentBackground = '#FFFFFF';
-  const colorText = legacyStyles.themeTextColor;
+  const colorText = legacyStyles.themeGrayDark;
```

Comment:
themeTextColor = themeGrayDark, so I got rid of `themeTextColor` to make it simpler.